doc/manuals/admin/README.md

```diff
@@ -0,0 +1,19 @@
+
+# Administration Guide
+
+Orion Context Broker: Administration Guide. 
+
+## Table of Contents
+
+* [Building from sources](build_source.md)
+* [Running Orion from command line](cli.md)
+* [Database administration](database_admin.md)
+* [Logs](logs.md)
+* [Watchdog](watchdog.md)
+* [Rush relayer](rush.md)
+* [Management REST inferface](management_api.md)
+* [Data model](database_model.md)
+* [Resources & I/O Flows](resources.md)
+* [Problem diagnosis procedures](diagnosis.md)
+* [Known limitations](known_limitations.md)
```

Comment:
The known_limitations.md file is currently under the user manual directory. I think there is no problem on considering it as part of the admin manual but, in that case, the known_limitations.md need to be move to the directory of that manual (otherwise, this link will fail).